doc/source/whatsnew/v1.4.0.rst

```diff
@@ -265,6 +265,7 @@ Groupby/resample/rolling
 Reshaping
 ^^^^^^^^^
 - :func:`concat` creating :class:`MultiIndex` with duplicate level entries when concatenating a :class:`DataFrame` with duplicates in :class:`Index` and multiple keys (:issue:`42651`)
+-Bug in :meth:`pandas.cut` on :class:`Series` with duplicate indices (:issue:`42185`) and non-exact :meth:`pandas.CategoricalIndex` (:issue:`42425`)
```

Comment:
this is failing the CI
> /home/runner/work/pandas/pandas/doc/source/whatsnew/v1.4.0.rst:282: WARNING: Bullet list ends without a blank line; unexpected unindent.

e.g. need a space after the '-'

and pls rebase once again CI should then be green.

Reply:
rebased. I don't think the doctest failures are related. Although still failing mypy typing

/home/runner/work/pandas/pandas/pandas/io/formats/style_render.py:649: DocTestFailure
/home/runner/work/pandas/pandas/pandas/io/formats/style_render.py:1254: UnexpectedException